src/components/interactive-ad.js

```diff
@@ -0,0 +1,122 @@
+import uuid from 'uuid';
+import { TruexAdRenderer } from '@truex/ctv-ad-renderer';
+
+// Use a random UUID for the "opt out of tracking" advertising id that is stable for a session.
+const optOutAdvertisingId = uuid.v4();
+
+// Exercises the True[X] Ad Renderer for interactive ads.
+export class InteractiveAd {
+    constructor(adBreak, videoController) {
+        const self = this;
```

Comment:
I think we don't actually use "self" anywhere but line 16 -- can probably get rid of it